src/styles/_variables.scss

```diff
@@ -18,3 +18,6 @@ $hacks-nav-inactive-color: #c5e0da;
 $hacks-text-color: #d2dad3;
 $hacks-dark-text-color: #52414c;
 $hacks-background-color: #1b263b;
+$hacks-a: #a6b7d6;
```

Comment:
Change these colors if you want

Reply:
colors are changed because I am paranoid